airflow/providers/exasol/hooks/exasol.py

```diff
@@ -216,22 +216,20 @@ def get_autocommit(self, conn) -> bool:
 
         :param conn: Connection to get autocommit setting from.
         :return: connection autocommit setting.
-        :rtype: bool
         """
         autocommit = conn.attr.get('autocommit')
         if autocommit is None:
             autocommit = super().get_autocommit(conn)
         return autocommit
 
     @staticmethod
-    def _serialize_cell(cell, conn=None) -> object:
+    def _serialize_cell(cell, conn=None) -> object:  # type: ignore
```

Comment:
What’s this ignoring?

Reply:
This is because of an incompatible signature in regards to the base class (returns type are incompatible with `DbApiHook._serialize_cell`. 

In my message up there (that was collapsed by github) I tried to enumerate these incompatibilities.

I think a `# type: ignore[override]` is more appropriate and explicit. I just updated that thanks. (Updated for all of them mysql, exasol and postgres).

Same for the get_permission, create_permission stated above. (compare to FAB base class)